bin/start_be.sh

```diff
@@ -87,6 +87,18 @@ fi
 
 export LD_LIBRARY_PATH=$STARROCKS_HOME/lib/hadoop/native:$LD_LIBRARY_PATH
 
+# check java version and choose correct JAVA_OPTS
+JAVA_VERSION=$(jdk_version)
+final_java_opt=$JAVA_OPTS
+if [[ "$JAVA_VERSION" -gt 8 ]]; then
+    if [ -z "$JAVA_OPTS_FOR_JDK_9" ]; then
```

Comment:
I have a question. why check jdk 9 here.

Reply:
> I have a question. why check jdk 9 here.

This pr is target at be support customizi java opts, so if users' java editions may be 1.8 or 1.9. JAVA_OPTS of different java versions may differ from each other, so script needs to check current JDK version to load the correct JAVA_OPTS in be.conf.